moments/Parsing.py

```diff
@@ -0,0 +1,1119 @@
+"""
+Functions for computing the SFS and related quantities from sequence data. 
```

Comment:
What are the "related quantities"? Expand docstring at top of document. Are these additional statistics, or what I think you mean: other things like total length from the bed file, stuff like that.